doozer/doozerlib/cli/images_streams.py

```
@@ -197,6 +210,14 @@ def mirror_image(cmd_start: str, upstream_dest: str):
                             f'Failed to mirror {upstream_entry_name}: {stderr}', (rc, stdout, stderr)
                         )
 
+                # #region agent log
+                _reg_keys = "N/A"
+                if registry_config_file:
+                    try:
+                        with open(registry_config_file) as _rf: _reg_keys = list(json.load(_rf).get("auths", {}).keys())
```

Ruff (0.15.18)

[error] Multiple statements on one line (colon)

(E701)